google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/mutaterows/MutateRowsAttemptCallable.java

```diff
@@ -148,12 +139,12 @@ public List<MutateRowsResponse> apply(Throwable throwable) {
     this.callContext = Preconditions.checkNotNull(callContext, "callContext");
     this.retryableCodes = Preconditions.checkNotNull(retryableCodes, "retryableCodes");
     this.retryAlgorithm = retryAlgorithm;
-    this.attemptSettings = retryAlgorithm.createFirstAttempt();
+    this.attemptSettings = retryAlgorithm.createFirstAttempt(null);
```

Comment:
I'm a bit confused why the arg was introduced here

Reply:
Reverted, as to not mix functional changes with random changes